src/AppBundle/Model/Entity/LDAP/PbnlAccount.php

```diff
@@ -0,0 +1,369 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: paul
+ * Date: 31.08.17
+ * Time: 00:17
+ */
+
+namespace AppBundle\Model\Entity\LDAP;
+
+use Ucsf\LdapOrmBundle\Annotation\Ldap\Attribute;
+use Ucsf\LdapOrmBundle\Annotation\Ldap\Dn;
+use Ucsf\LdapOrmBundle\Annotation\Ldap\ObjectClass;
+use Ucsf\LdapOrmBundle\Annotation\Ldap\SearchDn;
+use Ucsf\LdapOrmBundle\Annotation\Ldap\UniqueIdentifier;
+use Ucsf\LdapOrmBundle\Entity\Ldap\InetOrgPerson;
+
+/**
+ * Represents a LDAPUser object class, which is a subclass of InetOrgPerson
+ *
+ * @ObjectClass("pbnlAccount")
+ * @UniqueIdentifier("givenName")
+ * @SearchDn("ou=people,dc=pbnl,dc=de")
+ * @Dn("givenName={{ entity.givenName }},ou={{ entity.ou }},ou=people,dc=pbnl,dc=de")
+ */
+class PbnlAccount extends InetOrgPerson
+{
+
+    protected $ou;
+
+    /**
+     * the username
```

Comment:
Am Anfang schreibt man groß ;-)
Unnötige Füllwörter und Artikel ("the") würde ich weglassen. Entwickler möge es gerne kurz und prägnant ;-)
Sinnvoll wäre vielleicht, zu dokumentieren, welcher Zeichenvorat (Groß/Kleibuchstaben mit/ohne Leerzeichen, Umlaute ja/nein) für dieses Attribut erlaubt ist und wofür das Attribut verwendet wird. Zeichenvorat könnte auch als Reg Ex angegeben sein. Inhaltlich ginge es hier um den Nickname oder ersatzweise den Vornamen.